package.json

```diff
@@ -108,12 +108,14 @@
     "@testing-library/jest-dom": "6.1.3",
     "@testing-library/react": "14.0.0",
     "@testing-library/user-event": "14.5.1",
+    "@types/d3-color": "3.1.1",
     "@types/jest": "29.5.5",
     "@types/qlik-engineapi": "12.67.14",
     "@types/react": "18.2.24",
     "@types/react-dom": "18.2.8",
     "@types/react-window": "1.8.6",
     "babel-eslint": "10.1.0",
+    "d3-color": "3.1.0",
```

Comment:
This is used to convert the color definition coming from Engine to a format the browser supports. That logic should at some point be moved to the util repo so that both tables can utilize it. But I consider it out-of-scope for this PR.